verification/cbmc/sources/make_common_data_structures.c

```diff
@@ -30,12 +30,14 @@
 #include <proof_helpers/proof_allocators.h>
 
 void ensure_alg_properties_attempt_allocation(struct aws_cryptosdk_alg_properties *const alg_props) {
-    size_t md_name_size;
-    alg_props->md_name = can_fail_malloc(md_name_size);
-    size_t cipher_name_size;
-    alg_props->cipher_name = can_fail_malloc(cipher_name_size);
-    size_t alg_name_size;
-    alg_props->alg_name = can_fail_malloc(alg_name_size);
+    if (alg_props) {
```

Comment:
You can move the allocation for `alg_props` inside this function. The will be in sync with other allocators too.

Reply:
Alright, I will move it once the other conversation about this function is resolved.

Reply:
@adpaco-aws is this resolved?

Reply:
?